chirpr.py

```diff
@@ -83,9 +86,17 @@ def login_page():
 def user_page(handle):
     uid = db_access.get_id(handle)
     my_followers = []
+    name = None
+    chirp_list = []
     if 'user' in session:
         my_followers = get_followers()
-    return render_template('user_page.html', handle=handle, uid=uid, follow_data=db_access.follow_data(uid), my_followers=my_followers, get_user=db_access.get_user)
+        user = session['user']
+        name = db_access.get_user(user)
+    if db_access.user_exists(uid) is True:
+        chirp_list = db_access.get_chirps(uid)
+        return render_template('user_page.html', markdown=markdown2.markdown, handle=handle, uid=uid, user_data=db_access.user_data(uid), my_followers=my_followers, get_user=db_access.get_user,chirps=chirp_list, name=name)
```

Comment:
This is OK for now, but passing so many things to the template isn't best. We will talk about how to better organize this next meeting.